apps/www/_blog/2023-03-01-geo-queries-with-postgis-in-ionic-angular.mdx

```diff
@@ -127,21 +127,21 @@ Those are some powerful calculations, and we can easily use them through the Pos
 ```sql
 -- create database function to find nearby stores
 create or replace function nearby_stores(lat float, long float)
-returns setof record
+returns table (id public.stores.id%TYPE, name public.stores.name%TYPE, description public.stores.description%TYPE, lat float, long float, dist_meters float)
 language sql
 as $$
-  select id, name, description, st_astext(location) as location, st_distance(location, st_point(long, lat)::geography) as dist_meters
+  select id, name, description, st_y(location::geometry) as lat, st_x(location::geometry) as long, st_distance(location, st_point(long, lat)::geography) as dist_meters
```

Comment:
The change here is to match the model defined here
https://github.com/supabase/supabase/blob/master/apps/www/_blog/2023-03-01-geo-queries-with-postgis-in-ionic-angular.mdx#L250